ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1387,6 +1387,12 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         setNeedsLayout()
         invalidateIntrinsicContentSize()
     }
+
+    /// Returns the UIView of the `accessoryView`.
+    @objc open func getAccessoryView() -> UIView? {
+        return accessoryTypeView
```

Comment:
Any reason not to just make `accessoryTypeView` have a public getter?

Reply:
It's not possible because the `TableViewCellAccessoryType` is internal class so we aren't allowed to expose it unfortunately! The other approach was to create a public getter of the internal `accessoryTypeView` UIView but this approach would seem less confusing to anyone wondering why we have 2 properties of the same thing